pywatts/modules/rolling_rmse.py

```diff
@@ -0,0 +1,86 @@
+from typing import Dict
+
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+from pywatts.core.base import BaseTransformer
+from pywatts.core.exceptions.input_not_available import InputNotAvailable
+from pywatts.utils._xarray_time_series_utils import _get_time_indeces
+
+
+class RollingRMSE(BaseTransformer):
```

Comment:
Maybe we could restructure the pyWATTS library to pyWATTS.metrics, pyWATTS.core, pyWATTS.models, ... to give the user some structure.

Reply:
Good idea, I open an issue